Packs/CommonPlaybooks/Playbooks/playbook-Dedup_-_Generic_v4_README.md

```diff
@@ -1,50 +1,60 @@
+| fieldExactMatch | Select the incident field name you want the script to query.  <br/>For example, if you select &amp;lt;Type&amp;gt;, the playbook will query against the database for all incidents with the same type as your current incident.<br/> Note: If you use comma-separated values, the operator between them will be \*AND\*. |  | Optional |
+| fieldsToDisplay | A comma-separated list of additional incident fields to display in the context output. These fields can be used later on for layouts or other states if needed.<br/>\(Those which will not be taken into account when computing similarity\) |  | Optional |
+| fromDate | The start date to filter incidents. Date format is the same as in the incidents query page, for example, "3 days ago", "1 month ago", "2019-01-01T00:00:00 \+0200"\). | 1 months ago | Optional |
+| limit | The maximum number of incidents to query and set to context data.<br/>Default is: 200 | 200 | Optional |
+| minimunIncidentSimilarity | Retain incidents with a similarity score greater than the MinimunIncidentSimilarity.<br/>Default: 0.2<br/>Value should be between 0 to 1 \[0=low similarity, 1=identical\] | 0.2 | Required |
 | similarTextField | A comma-separated list of incident text fields to take into account when computing similarity. For example commandline, URL |  | Required |
 | CloseSimilar | Defines the threshold of similarity to close a similar incident. All similar incidents with similarity above this value will be closed.<br/>For example, if CloseSimilar is set to .8 and an incident has a similarity score of .9, the incident will be closed.<br/>The value should be between 0 and 1 \[0=low similarity , 1=identical\]. |  | Optional |
-| showIncidentSimilarityForAllFields | Whether to display the similarity score for each of the incident fields that were entered in the **similarTextField**. | True | Optional |
-| query | The argument for dedicated query on incidents. This helps reduce the query size.<br/> | -status:closed -category:job <br/>(Same as in the **Incident** tab). | Optional |
+| showIncidentSimilarityForAllFields | Whether to display the similarity score for each of the incident fields that were entered in the "similarTextField".<br/>Default: True | True | Optional |
+| query | The argument for dedicated query on incidents. This helps reduce the query size.<br/>Default \(same is in the Incident tab\): "-status:closed -category:job " | -status:closed -category:job | Optional |
 | closeReason | Specify the reason for closing the incident. This information will be added as a note/comment to the closed incident. | Closed by Dedup Playbook within inc ${incident.id} | Optional |
 
 ## Playbook Outputs
+
 ---
 
 | **Path** | **Description** | **Type** |
 | --- | --- | --- |
 | DBotFindSimilarIncidents | Return all the results from the "DBotFindSimilarIncidents" script. | string |
```

Comment:
This should be unknown right? returns a dict and not a string if I'm not mistaken.
For both outputs.

Reply:
changed the outputs type to 'unknown'